### PR TITLE
Add STDCI

### PR DESCRIPTION
**Pull request by arachmani**
Add STDCI and run the basic functional test on
pull requests

Signed-off-by: Asaf Rachmani <arachman@redhat.com>

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by arachmani** [comment]
ci test please

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by arachmani** [comment]
ci test please

File: automation/check-patch.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+main() {
+    TARGET="$0"
+    TARGET="${TARGET#./}"
+    TARGET="${TARGET%.*}"
+    TARGET="${TARGET#*.}"
+    echo "TARGET=$TARGET"
+    export TARGET
+
+    echo "Run functional tests"
+    exec automation/test.sh
```
Comment (rthallisey) [inline automation/check-patch.sh]: why use `exec` here instead of running `automation/test.sh` in the same shell?

Comment (arachmani) [reply automation/check-patch.sh]: To avoid the creation of a new process in the process tree

**Pull request merged**